### PR TITLE
SLO for CA in FAQ

### PR DESCRIPTION
**Pull request by mwielgus**

### CODE REVIEW

**Comment by MaciekPytel** [comment]
I feel this is formal enough that we should explicitly say that this SLO does not hold if you (over)use pod affinity / antiaffinity.

**Comment by aleksandra-malinowska** [comment]
How does request for CA pod factor into this?

**Comment by mwielgus** [comment]
Added info about affinity and pod requests/master node size.

File: cluster-autoscaler/FAQ.md [hunk 1]
```diff
@@ -115,6 +116,34 @@ Sice version 1.0.0 we consider CA as GA. It means that:
...
+from the time a pod is marked as unschedulable (by K8S scheduler) to the time CA issues scale-up request to the cloud provider
+(assuming that it is permitted). During our scalability tests (described 
+[here](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/proposals/scalability_tests.md)) we aimed at 
+max 20sec latency, even in the big clusters. We reach these goals on GCE on our test cases, however in practice the 
+performance may differ. Hence, users should expect:
+
+* No more than 30 sec latency on small clusters (less than 100 nodes with up to 30 pods each), with the average latency less in around 5 sec.
+* No more than 60 sec latency on big clusters (more than 100 nodes but less than 1000), with average latency around 15 sec.
+
+Please note, that the above performance can be achieved only in NO pod affinity and antiaffinity is used on any of the pods.
```
Comment (wasylkowski) [inline cluster-autoscaler/FAQ.md]: s/in/if/

File: cluster-autoscaler/FAQ.md [hunk 2]
```diff
@@ -115,6 +116,34 @@ Sice version 1.0.0 we consider CA as GA. It means that:
...
+Unfortunatelly the current implementation of the affinity scheduler predicate is about 3 orders of magnitude slower than
```
Comment (wasylkowski) [inline cluster-autoscaler/FAQ.md]: s/Unfortunatelly/Unfortunately/

File: cluster-autoscaler/FAQ.md (hunk 1 above)
Comment (mwielgus) [reply cluster-autoscaler/FAQ.md]: Done.

File: cluster-autoscaler/FAQ.md (hunk 2 above)
Comment (mwielgus) [reply cluster-autoscaler/FAQ.md]: Done.

**Pull request merged**